<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonOpenFile">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>510</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetBookings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>461</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDontModify1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Flightbooking</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDontModify2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rentalcar Reservation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDontModify3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hotelbooking</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelId">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditId">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPrice">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPrice">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFromDestination">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>From Destination:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelToDestination">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>To Destination:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAirline">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Airline</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFromDestination">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditToDestination">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAirline">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFlightArrival">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Arrival</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFlightDeparture">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Departure</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditFlightArrival">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditFlightDeparture">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>220</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditHotelTo">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>340</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditHotelFrom">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHotelName">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hotel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHotelTo">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>340</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHotelFrom">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHotelTown">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Town:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHotelName">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>270</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHotelTown">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
